app/ui/components/recruitmentHomePage.tsx

```diff
@@ -33,96 +35,69 @@ export default function RecruitementHomePage() {
-                  src="recruitement-image-1.png"
-                  justifyItems="center"
-                  alignItems="center"
-                />
-              </Center>
-              <Text
-                fontSize="3xl"
-                py="6"
-                fontFamily="Inter"
-                fontWeight="600"
-                lineHeight="120%"
-              >
-                1. Discover
-              </Text>
-              <Text
-                fontSize="xl"
-                fontFamily="Inter"
-                fontWeight="200"
-                lineHeight="160%"
-              >
-                Trouver facilement des compétences pointues pour vos projets et
-                contactez
-              </Text>
-            </GridItem>
-          </WrapItem>
-          <WrapItem>
-            <GridItem p="4" w="275px" h="100%">
-              <Center>
-                <Image
-                  alt="quelqu'un devant de la nouriture"
-                  src="recruitement-image-2.png"
-                  justifyItems="center"
-                  alignItems="center"
-                />
-              </Center>
-              <Text
-                fontSize="3xl"
-                py="6"
-                fontFamily="Inter"
-                fontWeight="600"
-                lineHeight="120%"
-              >
-                2. Copy & Paste
-              </Text>
-              <Text
-                fontSize="xl"
-                fontFamily="Inter"
-                fontWeight="200"
-                lineHeight="160%"
-              >
-                Travaillez main dans la main : décidez ensemble des modalités de
-                la mission puis collaborez en direct
-              </Text>
-            </GridItem>
-          </WrapItem>
-          <WrapItem>
-            <GridItem p="4" w="275px" h="100%">
-              <Center>
-                <Image
-                  alt="quelqu'un jouant de la guitare"
-                  src="recruitement-image-3.png"
-                  justifyItems="center"
-                  alignItems="center"
-                />
-              </Center>
-              <Text
-                fontSize="3xl"
-                py="6"
-                fontFamily="Inter"
-                fontWeight="600"
-                lineHeight="120%"
-              >
-                3. Export & Share
-              </Text>
-              <Text
-                fontSize="xl"
-                fontFamily="Inter"
-                fontWeight="200"
-                lineHeight="160%"
-              >
-                Recommencez. La mission est terminée? Débloquez le paiement en
-                un clic et partez à la recherche de votre prochain talent
-              </Text>
-            </GridItem>
-          </WrapItem>
+          <GridRecruitement
+            gridPadding="4"
+            gridWidth="275px"
+            gridHeight="100%"
+            imgAlt="quelqu'un devant son téléphone"
```

Comment:
This is a really bad Alt tbh, I wont require any fixes for it now but please keep in mind to keep it a good quality

Reply:
This is valable for every alt so far, I'll request changes for PR on pre-prod